portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,42 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Austin Teshuba</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Austin Teshuba</h1>
+
+      <img class="img-primary" src="/images/headshot.jpeg" alt="Profile Picture">
+
+      <p>Hi! Welcome to my portfolio.</p>
+
+      <a href="https://github.com/austinteshuba" target="_blank">Github</a>
+
+      <p> I'm a rising 3rd-year software engineering student at Western University.</p>
+
+      <p>
+          I’m also a technologist knowledgeable in web development with Angular, 
+          iOS development with Swift, data analysis and ML with Python, algorithms, 
+          object oriented design, and much more.
+      </p>
+
+      <p>
+          Most importantly, I’m a curious person and a lightening fast learner. 
+          Whether I’m mastering web development to complete a pressing project at Deloitte, 
+          or figuring out Twilio and Microsoft Azure in 24 hours at MHacks, 
+          I always approach new problems with a positive attitude and relentless determination.
+      </p>
+
+      <p> If you're solving big problems with even bigger impacts, please reach out!
```

Comment:
nit: `<p>` and content in the same line, but at line 27 there's a line break before the content. Try to follow the same pattern throughout the file

Reply:
Fixed!